src/domain/challenge/challenge/challenge.entity.ts

```diff
@@ -52,30 +49,31 @@ export class Challenge extends BaseEntity implements IChallenge, IGroupable {
     nullable: true,
     description: 'The shared understanding for the challenge',
   })
-  @OneToOne(() => Context, { eager: true, cascade: true })
+  @OneToOne(() => Context, { eager: false, cascade: true })
```

Comment:
It's OK to be eager loaded if it's one-to-one - won't have a huge performance hit.

Reply:
Made Community all eager loaded - loading stops by groups.